docs/config.md

```diff
@@ -38,9 +38,12 @@ The explanation and default value of each configuration item are as follows:
       # runtime_engine is the name of the runtime engine used by containerd.
       runtime_engine = ""
 
-      # runtime_root is the directory used by containerd for runtime state.
+      # runtime_root is the directory used by containerd for persistent runtime state.
       runtime_root = ""
 
+      # runtime_state is the directory used by containerd for volatile runtime state.
+      runtime_state = ""
```

Comment:
Why do we need this? We are not using it.

Reply:
There is no such thing in runc options, either.
https://github.com/containerd/containerd/blob/856ea51c8b5923b88ae41c9df501909c07776ea7/linux/runctypes/runc.pb.go#L40

Reply:
yeah it's a todo..